### ISSUE TITLE
Videos can't be played on loop

### ISSUE THREAD

**Issue by slime73** [love2d/love#1137]
**[Original report](https://bitbucket.org/rude/love/issue/1132) by William Dyce (Bitbucket: [wilbefast](https://bitbucket.org/wilbefast), GitHub: [wilbefast](https://github.com/wilbefast)).**

----------------------------------------

Hi there,

It's awesome to have video playback in 0.10.0 =) But as mentioned in my [tweet ](https://twitter.com/wilbefast/status/695018142469373953) I had a hard time getting videos to loop.

The trouble is that ``isPlaying`` return true even after the video has finished. I was able to implement looping using ``tell`` instead, but I needed to figure out the exact duration of the videos by hand and hard-code it into the demo because there isn't any method for getting the duration of a video.

Long story short any of the following would fix the issue:

* some kind of ``setLooping`` type function
* some kind of ``getDuration`` type function
* ``isPlaying`` could return false if the video has ended
* as an alternative: some kind of ``getStatus`` type function (which returns "ended" when the video has finished)
